### PR TITLE
[MO] Remove `_output_shape` nodes attribute while loading TF meta graph

### PR DESCRIPTION
**Pull request by achetver**
Root cause analysis: We faced an issue while loading some TF from .meta files when installed TF module version >= 2.2 (similar issue: https://github.com/davidsandberg/facenet/issues/1112). It is some inconsistency between node output shape and `_output_shape` attribute value and it leads to internal TF error 

Solution: Remove `_output_shape` nodes attribute while loading .meta graph to avoid inconsistency. After that the model can be successfully load to TF and converted to IR by MO.

Ticket: 60526

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR
* [x]  Transformation preserves original framework node names
* [x]  Transformation preserves tensor names

Validation:
* [x]  Unit tests: N/A - code will be covered by e2e tests;
* [x]  Framework operation tests: N/A - No new enabled operations;
* [x]  Transformation tests: N/A - No new implemented transformations;
* [x]  Model Optimizer IR Reader check: N/A.

Documentation:
* [x]  Supported frameworks operations list: N/A - No new supported operations;
* [x]  Supported **public** models list: N/A - Nothing to add;
* [x]  User guide update: N/A - Nothing to update.

### CODE REVIEW

File: tools/mo/openvino/tools/mo/front/tf/loader.py [hunk 1]
```diff
@@ -210,6 +210,9 @@ def load_tf_graph_def(graph_file_name: str = "", is_binary: bool = True, checkpo
         if not graph_file_name and meta_graph_file:
             meta_graph_file = deducing_metagraph_path(meta_graph_file)
             input_meta_graph_def = read_file_to_graph_def(tf_v1.MetaGraphDef(), meta_graph_file, is_binary)
+            for n in range(len(input_meta_graph_def.graph_def.node)):
+                if '_output_shapes' in input_meta_graph_def.graph_def.node[n].attr:
+                    del input_meta_graph_def.graph_def.node[n].attr['_output_shapes']
```
Comment (jane-intel) [inline tools/mo/openvino/tools/mo/front/tf/loader.py:215]: Shouldn't this work?
```suggestion
            for node in input_meta_graph_def.graph_def.node:
                if '_output_shapes' in node.attr:
                    del node.attr['_output_shapes']
```

Comment (achetver) [reply tools/mo/openvino/tools/mo/front/tf/loader.py:215]: Checked and yes, it works and looks not as complicated as my variant.

**Comment by achetver** [comment]
@yekruglov Please, take a look at this PR

**Comment by yekruglov** [comment]
> @yekruglov Please, take a look at this PR

I suggest to wait for the end of the CI run, because this attribute is used in some TF extractors

**Review by yekruglov — APPROVED** [review]

File: tools/mo/openvino/tools/mo/front/tf/loader.py [hunk 2]
```diff
@@ -210,6 +210,9 @@ def load_tf_graph_def(graph_file_name: str = "", is_binary: bool = True, checkpo
         if not graph_file_name and meta_graph_file:
             meta_graph_file = deducing_metagraph_path(meta_graph_file)
             input_meta_graph_def = read_file_to_graph_def(tf_v1.MetaGraphDef(), meta_graph_file, is_binary)
+            for node in input_meta_graph_def.graph_def.node:
```
Comment (mvafin) [inline tools/mo/openvino/tools/mo/front/tf/loader.py:213]: Please add comments for this code, as descriptive as possible

**Comment by mvafin** [comment]
Please run e2e full scope for tf models

File: tools/mo/openvino/tools/mo/front/tf/loader.py (hunk 2 above)
Comment (achetver) [reply tools/mo/openvino/tools/mo/front/tf/loader.py:216]: Added.

**Review by rkazants — APPROVED** [review]

**Pull request merged**